src/modules/google-organization-kp/includes/Rest_Controller.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+namespace Wordlift\Modules\Google_Organization_Kp;
+
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use Wordlift_Configuration_Service;
+
+class Rest_Controller {
+
+    public function register_hooks() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	public function register_routes() {
+        register_rest_route(
+			WL_REST_ROUTE_DEFAULT_NAMESPACE,
+			'/wl-google-organization-kp/pages',
+			array(
+				'methods'             => 'GET',
+				'callback'            => array( $this, 'list_pages' ),
+				'permission_callback' => '__return_true',
+			)
+		);
+
+		register_rest_route(
+			WL_REST_ROUTE_DEFAULT_NAMESPACE,
+			'/wl-google-organization-kp/countries',
+			array(
+				'methods'             => 'GET',
+				'callback'            => array( $this, 'list_countries' ),
+				'permission_callback' => '__return_true',
+			)
+		);
+
+		register_rest_route(
+			WL_REST_ROUTE_DEFAULT_NAMESPACE,
+			'/wl-google-organization-kp/field-data',
+			array(
+				'methods'             => 'GET',
+				'callback'            => array( $this, 'existing_data' ),
+				'permission_callback' => '__return_true',
+			)
+		);
+    }
+
+    public function list_pages() {
+		$pages = get_pages();
+
+		// Send back a 404 if no pages exist.
+		if ( empty($pages) ) {
+			return new \WP_Error(
+				'wl_google_organization_kp_data_not_found',
+				__( 'No data was found.', 'wordlift' ),
+				array(
+					'status' => 404,
+				)
+			);
+		}
+
+		// Return an array of pages with ID and title
+		$data = array();
+
+		foreach( $pages as $page ) {
+			$data[] = array(
+				'id'	=> $page->ID,
+				'title' => $page->post_title
+			);
+		}
+
+        return rest_ensure_response( $data );
+    }
+
+	public function list_countries() {
+		// @todo: Perhaps this should be moved elsewhere?
+		$countries = array(
```

Comment:
We already have them here src/includes/class-wordlift-countries.php

PS: I usually look for strings in code, e.g. 'Afghanistan' to see if we already did something related.

Reply:
I had a feeling it already existed somewhere. I should have thought to search for a country string! I've changed it to used this class now.